app/rql.py

```diff
@@ -56,6 +56,17 @@ class OrganizationRules(ModelRQLRules, AuditableMixin):
     status = FieldRule()
 
 
+class EntitlementRules(ModelRQLRules, AuditableMixin):
+    __model__ = Entitlement
+
+    id = FieldRule()
+    name = FieldRule()
+    datasource_id = FieldRule()
+    status = FieldRule()
+    redeemed_by = RelationshipRule(rules=OrganizationRules(), alias="events.redeemed.by")
+    terminated_by = RelationshipRule(rules=ActorRules(), alias="events.terminated.by")
```

Comment:
Good addition! Could you please add also `terminated_at = FieldRule(alias="events.terminated.at")` ?